src/SDK/MonitoringApplianceEmulator/ViewModels/ChartPropertyControlViewModel.cs

```diff
@@ -0,0 +1,130 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="ChartPropertyControlViewModel.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace Microsoft.Azure.Monitoring.SmartDetectors.MonitoringApplianceEmulator.ViewModels
+{
+    using System;
+    using System.Globalization;
+    using System.Windows.Media;
+    using LiveCharts;
+    using LiveCharts.Configurations;
+    using LiveCharts.Wpf;
+    using Microsoft.Azure.Monitoring.SmartDetectors.MonitoringApplianceEmulator.Models.Chart;
+    using Microsoft.Azure.Monitoring.SmartDetectors.RuntimeEnvironment.Contracts;
+
+    /// <summary>
+    /// The view model class for the <see cref="ChartPropertyControlViewModel"/> control.
+    /// </summary>
+    public class ChartPropertyControlViewModel : ObservableObject
```

Comment:
>ChartPropertyControlViewModel [](start = 17, length = 29)

I'll add UTs for this view model in later phase (task 1380566)

Reply:
why? please don't add code without test coverage

---
In reply to: [230673757](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/17#discussion_r230673757) [](ancestors = 230673757)